docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p><strong>Brennan:</strong></p><blockquote>var description: String ?</blockquote><blockquote><br></blockquote><blockquote>Maybe change this to "makeNoise" or something to communicate the side-effect nature?</blockquote><p><strong>Bart:</strong></p><blockquote>It doesn't print and returns a string without modifying the struct, so what would the side effect be?<span class="ql-cursor">﻿</span></blockquote>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>